cluster/gce/trusty/configure-helper.sh

```diff
@@ -498,10 +494,7 @@ start_kube_controller_manager() {
   if [ -n "${CONTROLLER_MANAGER_TEST_LOG_LEVEL:-}" ]; then
     log_level="${CONTROLLER_MANAGER_TEST_LOG_LEVEL}"
   fi
-  params="${params} ${log_level}"
-  if [ -n "${CONTROLLER_MANAGER_TEST_ARGS:-}" ]; then
-    params="${params} ${CONTROLLER_MANAGER_TEST_ARGS}"
-  fi
+  params="${params} ${log_level} ${CONTROLLER_MANAGER_TEST_ARGS:-}"
```

Comment:
We can move ${CONTROLLER_MANAGER_TEST_ARGS:-} to line 477, but ${log_level} is still calculated by lines 493--496. So we will still need line 497. Will update it.